doc/modules/classes.rst

```diff
@@ -1212,7 +1212,7 @@ Model validation
 
 .. _multiclass_ref:
 
-:mod:`sklearn.multiclass`: Multiclass and multilabel classification
+:mod:`sklearn.multiclass`: Multiclass classification
```

Comment:
I understand this is WIP but for the final version, please make sure that the underlines `===...` match the title length